ocp_resources/migration_policy.py

```diff
@@ -0,0 +1,59 @@
+from ocp_resources.resource import Resource
+
+
+class MigrationPolicy(Resource):
+    """
+    MigrationPolicy object
+    """
+
+    api_group = Resource.ApiGroup.MIGRATIONS_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        allow_auto_converge=None,
```

Comment:
Please add a docstring describe the args

Reply:
added